tools/topology/sof/pipe-kfbm-capture.m4

```diff
@@ -12,11 +12,37 @@ include(`buffer.m4')
 include(`pcm.m4')
 include(`dai.m4')
 include(`kpbm.m4')
+include(`mixercontrol.m4')
+include(`bytecontrol.m4')
 include(`pipeline.m4')
 
 dnl Soun Trigger Stream Name)
 define(`N_STS', `Sound Trigger Capture '$1)
 
+#
+# Controls
+#
+
+# kpbm initial parameters
+CONTROLBYTES_PRIV(KPB_priv,
```

Comment:
what are these ? what structure, are they real coefficients or test data ???

Reply:
@lgirdwood we are aligned to use private kcontrol bytes for kpb/Intel KPD initialization(the corresponding structs are sof_kpb_config and sof_detect_test_config), maybe I should add these info into comment messages.

Reply:
yes please as other users will use this as an **example**

Reply:
done.